### PR TITLE
nonship Jakarta NoSQL/Data features

### PR DESCRIPTION
**Pull request by njr-11**
Create a nonship Jakarta NoSQL feature by pulling in the latest Jakarta NoSQL artifacts from Maven and using the transformer to switch them to jakarta package names.  This will help us in exploring how the design of Jakarta Data can interact with and reuse from the Jakarta NoSQL API.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
#build

**Comment by helyarp** [comment]
I added the **ID review not needed** label because the .properties files have no messages to review. The strings in the .properties files look fine.

**Comment by posmith** [comment]
L2 message review not required

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

File: dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/Delete.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *******************************************************************************/
+package jakarta.data;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotates a repository method with to designate it as a delete operation.
```
Comment (nmittles) [inline dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/Delete.java]: Check grammar.

File: dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/Delete.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * The <code>@Where</code> annotation can be added to provide conditions.
+ * Do not combine on a single method with {@link Update &#64;Update} or with
+ * {@link Query &#64;Query}, which is can be more advanced way of providing
```
Comment (nmittles) [inline dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/Delete.java:22]: Check grammar.

**Review by nmittles — APPROVED** [review]

**Comment by njr-11** [comment]
Fixed review comment and a bundle name typo.
#build
#libby

**Comment by njr-11** [comment]
Rebased on to latest and fixed issue with a nonship dummy feature that we only need to allow transform so that the unused original bundle won't generate complaints
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
The build needed to have 1 child FAT resubmitted due to what looks like an infrastructure issue, and then the resubmit got stuck but in a different place, but otherwise looks good.

**Pull request merged**